test/specs/js-file.js

```diff
@@ -8,27 +8,43 @@ var fs = require('fs');
 describe('modules/js-file', function() {
 
     function createJsFile(sources, options) {
-        return new JsFile(
-            'example.js',
-            sources,
-            JsFile.parse(sources, esprima),
-            options || {}
-        );
+        var params = {
+            filename: 'example.js',
+            source: sources,
+            esprima: esprima
+        };
+
+        for (var i in options) {
```

Comment:
We already have lodash.assign (https://github.com/jscs-dev/node-jscs/blob/mdevils.jsfile-constructor/package.json#L71), maybe use it here?